test/compilable/test18199.d

```diff
@@ -0,0 +1,73 @@
+// REQUIRED_ARGS:
+// PERMUTE_ARGS:
```

Comment:
Those two lines aren't required. On the contrary, `PERMUTATE_ARGS` shouldn't be specified manually (except if there's a good reason to do so)

Reply:
remove, thank you